inc/pantheon-multisite-finalize.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * If a site has multisite enabled, but has not had the final installation
+ * steps completed, alert the user and provide links.
+ */
+
+function pantheon_multisite_install_finalize_message() {
+	?>
+	<div class="notice notice-info is-dismissible">
+		<p>Your WordPress Site Network is almost ready!</p>
```

Comment:
Let's [escape](https://developer.wordpress.org/apis/security/escaping/) this output, for example:

```suggestion
		<p><?php echo esc_html( 'Your WordPress Site Network is almost ready!' ); ?></p>
```

Reply:
better yet 
```suggestion
		<p><?php esc_html_e( 'Your WordPress Site Network is almost ready!', 'pantheon' ); ?></p>
```

(not actually sure what the textdomain of the mu plugin is